doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -561,7 +561,7 @@ might be useful in understanding the format.
 The Simple Query Language provides a simplified syntax to retrieve entities which match a set of
 conditions.
 A query is composed by a list of statements separated by the ';' character.
-Each statement express a matching condition.
+Each statement expresses a matching condition.
```

Comment:
I see three choices:
1. Maintain this kind of fixes in a separate file - to not forget 
2. Change also in the freezed (frozen? :-) doc
3. Live with the noise

Easiest option: 3 (I don't see that this noise will bother too much - of course, bigger changes should not be made - just simple typos)

I would put a 'weak vote' for option 3